conf/application.conf

```diff
@@ -58,4 +58,6 @@ microservice {
 
 tracking-consent-frontend {
   gtm.container = "d"
-}
\ No newline at end of file
+}
+
+contact-frontend.serviceId = "Service Availability"
```

Comment:
if it talks to `contact-frontend` then do we need its host/port config or not required ?

Reply:
there is a `contact-frontend.url` field which can be set to `http://localhost:9250` but that's already the default.

Reply:
I think I need to raise a PR in all environments though:
`microservice.services.contact-frontend.host: "www.qa.tax.service.gov.uk"`

Reply:
hmm not sure, check their readme .. 
may be `play-frontend-hmrc` can do the url construction for us without us needing to add the url to each env

Reply:
spoke to plats-ui, they're saying it's all resolved internally https://github.com/hmrc/play-frontend-hmrc/blob/main/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcReportTechnicalIssueHelper.scala.html#L29C19-L29C40